### PR TITLE
[WIP] Develop beta

### PR DESCRIPTION
**Pull request by benhylau**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,3 +131,56 @@ To add a new module, use **scripts/ipfs/** as an example to:
 * Your computer can be a node too! It will mesh with the Pi's over your router. See the [cjdns repository](https://github.com/cjdelisle/cjdns) on how to set this up.
 
 * Original plan for this repository and early benchmark results are available in [the doc folder](https://github.com/tomeshnet/prototype-cjdns-pi/blob/master/docs/).
+
+## FAQ
+
+**Q:** Why do my Orange Pi Zero USB headers not work?
+
+**A:** Some images are missing the USB overlay.  Simply add the following to the /boot/armbianEnv.txt file and restart the pi.
+```overlays=usbhost2 usbhost3```
```
Comment (benhylau) [inline README.md]: For inline, use single `. For code block, you must have the three backticks in its own line. Please see other parts of this doc.

File: README.md [hunk 2]
```diff
@@ -127,3 +131,56 @@ To add a new module, use **scripts/ipfs/** as an example to:
 * Your computer can be a node too! It will mesh with the Pi's over your router. See the [cjdns repository](https://github.com/cjdelisle/cjdns) on how to set this up.
 
 * Original plan for this repository and early benchmark results are available in [the doc folder](https://github.com/tomeshnet/prototype-cjdns-pi/blob/master/docs/).
+
+## FAQ
+
+**Q:** Why do my Orange Pi Zero USB headers not work?
+
+**A:** Some images are missing the USB overlay.  Simply add the following to the /boot/armbianEnv.txt file and restart the pi.
```
Comment (benhylau) [inline README.md]: We bold dirs in this repo. See: https://github.com/tomeshnet/prototype-cjdns-pi/pull/123/files#diff-04c6e90faac2675aa89e2176d2eec7d8R126

Comment (benhylau) [inline README.md]: One space after period.

File: README.md [hunk 3]
```diff
@@ -127,3 +131,56 @@ To add a new module, use **scripts/ipfs/** as an example to:
...
+**Q:** Why do I get an error about a locked file when I try to install the node on an Orange Pi?
+
+**A**: The daily apt upgrade sometimes starts up in the background locking the APT database.  This will cause the script to fail as it tries to install the required software. Wait for the upgrade to finish.
+
+
+**Q:** Can I connect a serial cable (ttl) to a Raspberry Pi (or Rock64)?
+
+**A:** Yes, there are TTL pins in the gpio pins. They are as follows  
+```
+    VCC → RPi Pin 02 (5V)
```
Comment (benhylau) [inline README.md]: Why are these indented?

File: contrib/node-exporter-ubiquity/install [hunk 4]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```
Comment (benhylau) [inline contrib/node-exporter-ubiquity/install]: Filename should be ubiquiti with an `i`

File: contrib/node-exporter-ubiquity/install [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+
+sudo cp "nodestats-tomesh.py" "/opt/tomesh/nodestats-tomesh.py"
+sudo cp "node-exporter-ubiquiti.service" /lib/systemd/system/node-exporter-ubiquiti.service
+sudo systemctl enable node-exporter-ubiquiti.service
+sudo systemctl start node-exporter-ubiquiti.service
+
+echo Remember to edit /opt/tomesh/nodestats-tomesh.py if your equipment ip address is different then 192.168.1.20
```
Comment (benhylau) [inline contrib/node-exporter-ubiquity/install]: `s/then/than`

File: scripts/prometheus-node-exporter/nodestats-tomesh.py [hunk 6]
```diff
@@ -0,0 +1,77 @@
+import os
```
Comment (benhylau) [inline scripts/prometheus-node-exporter/nodestats-tomesh.py]: Tabs in this file should be spaces.

File: scripts/install2 [hunk 7]
```diff
@@ -101,8 +119,19 @@ if [ -z "$WITH_MESH_POINT" -o "$WITH_MESH_POINT" != "true" -a "$WITH_MESH_POINT"
     else
         echo -e "\e[1;32mMesh Point interface will be configured\e[0m"
         WITH_MESH_POINT=true
+        read -p "Continue with 802.11s instead of older AdHoc (IBSS)?  (Y/n)? " -n 1 -r
```
Comment (benhylau) [inline scripts/install2]: I think we are presenting the terms `Mesh Point`, `802.11s`, and `AdHoc (IBSS)` in a confusing way here.

The question text is also inconsistent with the rest of the questions.

File: scripts/install2 [hunk 8]
```diff
@@ -348,5 +400,7 @@ echo "" >> ~/.bash_profile
 echo "# print mesh node status" >> ~/.bash_profile
 echo "status" >> ~/.bash_profile
 
+echo -e "Your nodes new hostname is $newhostname"
```
Comment (benhylau) [inline scripts/install2]: node's

**Comment by benhylau** [comment]
@darkdrgn2k Please address current comments and check for typo and format consistency before requesting review.

File: scripts/prometheus-node-exporter/nodestats-tomesh.py (hunk 6 above)
Comment (darkdrgn2k) [reply scripts/prometheus-node-exporter/nodestats-tomesh.py]: ```cat scripts/prometheus-node-exporter/nodestats-tomesh.py  | grep -P '\t' ```

Returns nothing. What tabs am I missing?

**Pull request merged**